### ISSUE TITLE
No Ethereum folder and no keystore

### ISSUE THREAD

**Issue by jaudo** [ethereum/mist#2719]
Version: Last version (Downloaded a week ago 1 july 2017)
OS & Version: windows 7
Node type: `eth/geth`
Number of blocks synchronized: 2 peers 1 965 221 blocks left

- I send ether to the wallet contract but it doesn't show up
When I create accounts, there's no Ethereum folder, no keystore, nothing!!!!

I transfered there some tokens, and it doesn't show up, I'm trying to export it but nothing!

I'm very dissapointed, I transfered there some tokens and know I'm unable to recover it!

**Comment by jaudo** [comment]
I tried to download the mist wallet in my laptop and it works. WTF is going on? I think I lost 90€ because of that. There's no Ethereum folder anywhere, no UTC file anywhere, yet this fucking program still let's me create accounts. 

My fault for trusting this software. Now there's nobody who can solve me that. Is there any command to run in geth.exe? Terrible.

**Comment by sathishvj** [comment]
The ether (if it's there in the account) will show up only when all blocks are synced.  In the meantime you can check your balance at etherscan.io/address/0x... Put your address at the end.

Depending on your platform, the ethereum related files are most likely to be at the locations below:
Mac: ~/Library/Ethereum
Linux: ~/.ethereum
Windows: %APPDATA%\Ethereum

**Comment by jaudo** [comment]
I downloaded another version of the wallet, it looks stable and has created the Ethereum folder in the APPDATA folder, but the old wallets have dissapeared, and now it's working correctly, so basically I lost my wallet because of a shitty buggy software, ¡that was the first choice in the ethereum official web page!

obviously ethereum is a very ambitious software, but it's full of errors since it hasn't been properly tested. I can't trust this software when the official wallet get's crazy.

Now it's impossible for me to recover the wallet, since I only have it's address and my password, but I don't have the private key since the buggy software didn't made it.

It's my fault too for not verifying the correct behavious of the wallet, but I'm, obviously, dissapointed.

This Ethereum project is very ambitious, but needs elite developers, and now I realized a 23 year old kid without college experience can't develop an elite software. So for me, ethereum is dead, good luck in the project anyways.

**Comment by ihatecreatingaccounts** [comment]
yeah, fuck ether. many people get fucked over by this "bug". the software has an option to skip password creation, so you DO NOT NEED A PASSWORD, yet it always will ask for one only to lock away your ether.
good thing is im living in the same country as the creator, so this faggot is in to be sued.

**Comment by antuerius** [comment]
Mea Culpa, yet ca. 2014, foolishly I tried installing the Ethereum wallet on an older version of computer than the ware can manage [mac 10.6 -- mac 10.8]. This caused a whack of problems, including a consequential system crash or two. With receiving the email with wallet, while part of it installed, I documented the receipt with btc issuing, transaction and public eth address; and I created and kept a good password. The transaction clearly shows on Etherscan.

Eth wallet updates never became legacy compatible. It was all frustrating and profitless. I followed along but could not participate.

I have tried a few passes of disk rescue, and disk drill, to discover the original files. I contacted my ISP to see if the original 2014 email can be documented. I also indulged the invitation to contact helpdesk at Eth; but the emails do not complete to their stated address.

I am reading these boards on possible solutions, and am amenable to any technical advice; but am left musing on whether my venture capital has vanished into the ether.

**Comment by antuerius** [comment]
A forensic laboratory told me they cannot be of any help unless there is physical damage to the computer. Plainly, why can't Ethereum reissue my wallet. When I made the investment, I recall being told it was possible to have the email sent a second time. Technically, feelings may not count; but there really is an intuitive factor here of having been cheated.

**Comment by antuerius** [comment]
Are there better data recovery wares? I invested in this somewhat goofy futurist enterprise and for all the genius claimed to be about, only my "luck" has gone into the Ether. For all my investment, Eth helpdesk is a halloween mask. 

You guys should do a schedule optimization study to determine the exact amount of time required to deliver short shrift to honest investors against technocratic 'karma' accusations. Is it three years; or more, just so as to fall on Christmas? On the advice given, I backed up the unusable Ethereum email files onto a second disk, and a usb key. In the interim three years of demoralizing forks, ever new wallet versions and poor investor support, both crashed or failed and/or got wiped. Expecting hardware to be stable for three years is a sort of 'karma' failure all its own. Discounting these real time experiences as merely emotional or irresponsible is incorrect, eth also degraded investor faith -- specifically through the culturally ugly media quips of globalist personage Vinay Gupta. 

At the level of legal proof, I have the document that Etherum.org sent, the ethersale-app file. This contains my public key, the blockchain trxn, my issuing wallet address, and the corresponding dates. I also have a password associated with the date of investment. Help desk remains ghoulishly unresponsive. 

I have read the Git boards and run a few passes of data rescue and disk drill on all devices, but so far, eth continues to own all my 'luck' and money.
